### PR TITLE
Login form

### PR DESCRIPTION
**Pull request by FarahZaqout**
adds the login form with its proper style and media query
#13

### CODE REVIEW

File: client/src/components/Login/Login.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  color: goldenrod;
+  transition: 1s;
+}
+
+.remember-me__span {
+  font-size: 13px;
+  margin: 15px 0% 0% 6px;
+}
+
+@media only screen and (min-width: 1000px) {
```
Comment (ashatat) [inline client/src/components/Login/Login.css]: My opinion is that we should skip media query in this project, it takes a lot of time, and I don't think it is priority

@amusameh , @FarahZaqout 
what do you think?

File: client/src/App.jsx [hunk 2]
```diff
@@ -1,28 +1,22 @@
...
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
-    );
+    return(
+      <Router>
+        <Switch>
+          <Route exact path="/my-account" >
```
[the remark this answers is not in the document]
Comment (FarahZaqout) [reply client/src/App.jsx]: this is how the website does it. the ```my-account``` endpoint has the login and signup forms together.

File: client/src/components/Login/Login.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+.sign-up-form__text {
+  margin: 30px 0% 12px 0%;
+  display: flex;
+  flex-direction: column;
+  background-color: inherit;
+  border-top: 0px;
+  border-right: 0px;
+  border-left: 0px;
+  border-color: #e5e5e5;
```
[the remark this answers is not in the document]
Comment (FarahZaqout) [reply client/src/components/Login/Login.css]: hm, I didn't realize you can do that. Will change it now.

File: client/src/components/Login/Login.css (hunk 1 above)
Comment (FarahZaqout) [reply client/src/components/Login/Login.css]: I am inclined to agree. I think we should make a judgment as to how quick we can progress with or without it. Surely, CSS is not the priority in this project. I don't mind skipping it.

**Pull request merged**